Project/Project/ViewController.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  ViewController.swift
+//  Project
+//
+//  Created by Ryan Murphy on 6/3/19.
+//  Copyright © 2019 Ryan Murphy. All rights reserved.
+//
+
+import UIKit
```

Comment:
You are not using this file, so you can delete it so that it doesn't take up space in your app.